application/_docker/database/scripts/01-ddl.sql

```diff
@@ -56,4 +56,59 @@ CREATE TABLE "LendingAssociatedNotificationTask" (
     "lendingId" BIGINT NOT NULL,
     "notificationTaskId" BIGINT NOT NULL,
     PRIMARY KEY ("lendingId", "notificationTaskId")
+);
+CREATE TABLE Loan
+(
+    "loanId"        BIGINT PRIMARY KEY,
+    "startDate"     DATE   NOT NULL,
+    "returnBy"      DATE,
+    "returnedOn"    DATE,
+    "loanStatus"    VARCHAR(50),
+    "membershipId"  BIGINT NOT NULL,
+    "productCopyId" BIGINT NOT NULL,
+    FOREIGN KEY ("membershipId") REFERENCES "Membership" ("membershipId") ON UPDATE CASCADE ON DELETE RESTRICT,
+    FOREIGN KEY ("productCopyId") REFERENCES "ProductCopy" ("productCopyId") ON UPDATE CASCADE ON DELETE RESTRICT
+);
+
+CREATE TABLE "Reservation"
+(
+    "reservationId"   BIGSERIAL PRIMARY KEY,
```

Comment:
Ik denk dat we binnen de database moeten zorgen dat alle Id´s van hetzelfde datatype zijn. Dit gaat waarschijnlijk niet fout, maar het ziet er inconsequent uit. Ik zie hier dat de loanId een bigint is en het reservationId een bigserial. Heb niet per se een voorkeur.

Daarnaast zie ik ook dat membershipId op Membership een bigserialis en de referentie naar datzelfde veld op Reservation een bigint is.

Reply:
Ah we hebben alle 3 wat anders gedaan her en der, ik maak er overal Bigserials van

Reply:
Ah good to know, thanks!